android/src/main/java/com/tailscale/ipn/ui/util/AndroidTVUtil.kt

```diff
@@ -15,7 +15,7 @@ import com.tailscale.ipn.ui.util.AndroidTVUtil.isAndroidTV
 object AndroidTVUtil {
   fun isAndroidTV(): Boolean {
     val pm = UninitializedApp.get().packageManager
-    return (pm.hasSystemFeature(PackageManager.FEATURE_TELEVISION) ||
+    return (pm.hasSystemFeature(@Suppress("deprecation") PackageManager.FEATURE_TELEVISION) ||
```

Comment:
add comment to migrate to FEATURE_LEANBACK pls

Reply:
We check both...  It's just for compatibility with old versions.